changelog/pattern-deprecate.dd

```diff
@@ -8,3 +8,57 @@ $(MREF std, regex) and $(MREF std, algorithm). They will be removed from
 $(MREF std, string) on May 2018.
 
 If you still need to use these, please see $(LINK2 https://github.com/dlang/undeaD, undeaD).
+
+Here's a simple guide on how to migrate your existing code.
```

Comment:
Avoid "Here's X" and adjectives describing the text itself ("simple"); consider: "Examples of the deprecated functions, and their modern replacements:"